src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -60,22 +69,22 @@
 			<div class="tw-col-span-2 tw-relative tw-grow">
 				<kv-section-modal-loader :loading="loading" bg-color="secondary" size="large" />
 				<div v-if="initialLoadComplete">
-					<loan-search-saved-search
-						v-if="enableSavedSearch && showSavedSearch && !savedSearchSuccess"
-						:loan-search-state="loanSearchState"
-						:all-facets="allFacets"
-						:show-success-message="showSavedSearchSuccessMessage"
-						:user-id="userId"
-					/>
 					<loan-search-filter-chips
 						:loan-search-state="loanSearchState"
 						:all-facets="allFacets"
 						@updated="handleUpdatedFilters"
 						@reset="handleResetFilters"
 					/>
-					<p class="tw-hidden lg:tw-block tw-mt-1">
+					<p class="tw-hidden lg:tw-inline-block tw-mt-1 tw-align-center">
 						{{ totalCount }} Loans
 					</p>
+					<loan-search-saved-search
+						class="tw-hidden lg:tw-inline-block tw-mt-1 tw-align-center tw-justify-self-auto"
+						v-if="enableSavedSearch && showSavedSearch && !savedSearchSuccess"
```

Comment:
`!savedSearchSuccess` can be removed from both instances here and above as it's now been removed and won't be defined.

Reply:
Fixed!